cmd/vic-machine/upgrade/upgrade.go

```diff
@@ -120,7 +120,14 @@ func (u *Upgrade) Run(clic *cli.Context) (err error) {
 		return err
 	}
 
-	op.Infof("### Upgrading VCH ####")
+	var action management.Action
+	if !u.Data.Rollback {
```

Comment:
can you remove negation and swap if/else blocks?

Reply:
I considered that. Usually I prefer to avoid negation, but in this case it seemed more intuitive to have the upgrade case first, in part for consistency with the rest of the file:

https://github.com/vmware/vic/blob/538f1de9bad0c14980faeba2d487337e11375d9a/cmd/vic-machine/upgrade/upgrade.go#L124

https://github.com/vmware/vic/blob/538f1de9bad0c14980faeba2d487337e11375d9a/cmd/vic-machine/upgrade/upgrade.go#L224